python/ray/data/_internal/execution/bulk_executor.py

```diff
@@ -35,7 +36,7 @@ def execute(
         assert not self._executed, "Can only call execute once."
         self._executed = True
         if not isinstance(dag, InputDataBuffer):
-            logger.info("Executing DAG %s", dag)
+            logger.get_logger().info("Executing DAG %s", dag)
```

Comment:
Wanted to confirm that this and the `debug()` a few lines below are intended to always write to both dataset logfile and stdout, as opposed to using `log_to_stdout=context.enable_auto_log_stats` to conform with the context configuration.

Reply:
This message is not stats so I think `enable_auto_log_stats` doesn't apply it.